plugin.go

```diff
@@ -87,6 +87,13 @@ func (c *Plugin) Run(cliConnection plugin.CliConnection, args []string) {
 			return config.Encrypt(cliConnection, configServerInstanceName, plainText, fileToEncrypt, authClient)
 		})
 
+	case "config-server-delete-git":
```

Comment:
I think we need to add this to `docs/generate-cli-docs-from-help.sh`

Reply:
Uh? How did I miss that? Thanks for catching this. Just updated the PR commit now.